src/proxy-ui-api/frontend/tests/e2e/pages/tabs/settingsTab.js

```diff
@@ -39,6 +41,61 @@ var settingsTabCommands = {
   },
 };
 
+var systemParametersCommands = {};
+
+var backupAndRestoreCommands = {
+  enterFilterInput: function (input) {
+    this.clearValue2('@searchField');
+    this.setValue('@searchField', input);
+    return this;
+  },
+  clearFilterInput: function () {
+    this.clearValue2('@searchField');
+    return this;
+  },
+  clickCreateBackup: function () {
+    this.click('@backupButton');
+    return this;
+  },
+  addBackupToInput: function (filepath) {
+    this.api.setValue('//input[@type="file"]', path.resolve(filepath));
+    return this;
+  },
+  clickDownloadForBackup: function (backupFilename) {
+    this.click(
+      'xpath',
+      `//table[contains(@class, "xrd-table")]/tr/td[text() = "${backupFilename}"]/..//button[contains(@data-test, "backup-download")]`,
+    );
+
+    return this;
+  },
+  clickRestoreForBackup: function (backupFilename) {
+    this.click(
+      'xpath',
+      `//table[contains(@class, "xrd-table")]/tr/td[text() = "${backupFilename}"]/..//button[contains(@data-test, "backup-restore")]`,
+    );
+    return this;
+  },
+  clickDeleteForBackup: function (backupFilename) {
+    this.click(
+      'xpath',
+      `//table[contains(@class, "xrd-table")]/tr/td[text() = "${backupFilename}"]/..//button[contains(@data-test, "backup-delete")]`,
+    );
+    return this;
+  },
+};
+
+var confirmationDialog = {
```

Comment:
Could maybe use `const`

Reply:
Agree. This and all other var declarations changed to const.